PixelKnot/src/main/java/info/guardianproject/pixelknot/StegoEncryptionJob.java

```diff
@@ -75,8 +75,14 @@ public void run() {
             addProcess(new Runnable() {
                 @Override
                 public void run() {
-                    Map.Entry<String, String> pack = Aes.EncryptWithPassword(getPassword(), mMessage, getPasswordSalt()).entrySet().iterator().next();
-                    mMessage = Constants.PASSWORD_SENTINEL.concat(new String(pack.getKey())).concat(pack.getValue());
+                    try {
+                        Map.Entry<String, String> pack = Aes.EncryptWithPassword(getPassword(), mMessage, getPasswordSalt()).entrySet().iterator().next();
+                        mMessage = Constants.PASSWORD_SENTINEL.concat(new String(pack.getKey())).concat(pack.getValue());
+                    } catch (Exception e) {
```

Comment:
We saw an OutOfMemoryException in the crash logs. But we should catch all and abort the job.